pylops/utils/multiproc.py

```diff
@@ -1,7 +1,15 @@
 import time
+from typing import List, Tuple
 
+import numpy.typing as npt
 
-def scalability_test(Op, x, workers=[1, 2, 4], forward=True):
+
+def scalability_test(
+    Op,
+    x: npt.ArrayLike,
+    workers: List = [1, 2, 4],
```

Comment:
maybe List[int]?

Reply:
So far I didnt add the [] to List and Tuple. I did it now... phew, so much work ;)